### ISSUE TITLE
Uncaught TypeError: Arguments to path.join must be strings

### ISSUE THREAD

**Issue by victorkane** [lloeki/ex-mode#40]
unamed file

using vim-mode, type:

````
i (insert)
one
two
<esc>
:w 
````

**Atom Version**: 0.199.0
**System**: Mac OS X 10.10.3
**Thrown From**: [ex-mode](https://github.com/lloeki/ex-mode) package, v0.4.1

### Stack Trace

Uncaught TypeError: Arguments to path.join must be strings

```
At /Users/victorkane/.atom/packages/ex-mode/lib/ex-state.coffee:56

TypeError: Arguments to path.join must be strings
  at Object.posix.join (path.js:449:13)
  at getFullPath (/Users/victorkane/.atom/packages/ex-mode/lib/ex.coffee:35:15)
  at Ex.write (/Users/victorkane/.atom/packages/ex-mode/lib/ex.coffee:131:20)
  at Ex.w (/Users/victorkane/.atom/packages/ex-mode/lib/ex.coffee:143:5)
  at /Users/victorkane/.atom/packages/ex-mode/lib/ex.coffee:1:1
  at Command.execute (/Users/victorkane/.atom/packages/ex-mode/lib/command.coffee:148:7)
  at ExState.processOpStack (/Users/victorkane/.atom/packages/ex-mode/lib/ex-state.coffee:49:17)
  at ExState.pushOperations (/Users/victorkane/.atom/packages/ex-mode/lib/ex-state.coffee:40:6)
  at ExViewModel.ViewModel.confirm (/Users/victorkane/.atom/packages/ex-mode/lib/view-model.coffee:12:14)
  at ExViewModel.module.exports.ExViewModel.confirm (/Users/victorkane/.atom/packages/ex-mode/lib/ex-view-model.coffee:35:5)
  at ExViewModel.confirm (/Users/victorkane/.atom/packages/ex-mode/lib/ex-view-model.coffee:1:1)
  at ExCommandModeInputView.module.exports.ExCommandModeInputView.confirm (/Users/victorkane/.atom/packages/ex-mode/lib/ex-command-mode-input-view.coffee:46:16)
  at atom-text-editor.<anonymous> (/Users/victorkane/.atom/packages/ex-mode/lib/ex-command-mode-input-view.coffee:1:1)
  at atom-text-editor.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
  at atom-text-editor.elemData.handle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:519:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:354:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:178:20)

```

### Commands

```
     -5:31.5.0 settings-view:open (atom-text-editor.editor.is-focused)
     -3:46 core:confirm (atom-text-editor.editor.mini.is-focused)
  8x -2:03.8.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -1:31 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:14 vim-mode:activate-insert-mode (atom-text-editor.editor.vim-mode.command-mode.is-focused)
  2x -0:10.7.0 editor:newline (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
     -0:08.9.0 vim-mode:activate-command-mode (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
     -0:07.7.0 ex-mode:open (atom-text-editor.editor.vim-mode.is-focused.command-mode)
     -0:02.1.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
atom-html-preview, v0.1.6
autocomplete-haskell, v0.2.1
color-picker, v1.7.0
ex-mode, v0.4.1
vim-mode, v0.46.0

# Dev
No dev packages
```

**Comment by jazzpi** [comment]
How did you encounter this error? Is it reproducible?

> [Enter steps to reproduce below]

**Comment by lloeki** [comment]
fixed by c43b2658
